### DISCUSSION TITLE
Can't use command 'compress'

### DISCUSSION THREAD

**Discussion by PPdavidXD** [deepmodeling/deepmd-kit#437] [Q&A]
Hi, I'm using the deepmd-kit 1.3.2 python interface downloaded from conda. I found that 'Compress' is not in the valid subcommands of dp, while I saw on deepmd.readthedocs.io there should be a command to compress the model.
Is that a problem of my dp version? Thank you very much.

**Comment by njzjz** [comment]
This feature is still in development -- meaning it is still in the `devel` branch. https://github.com/deepmodeling/deepmd-kit/tree/devel

**Reply by PPdavidXD** [reply]
Thank you for your reply. Hope to see this feature coming soon.
